etcdmain/etcd.go

```diff
@@ -216,15 +220,61 @@ func startProxy(cfg *config) error {
 		return err
 	}
 
-	// TODO(jonboulle): update peerURLs dynamically (i.e. when updating
-	// clientURLs) instead of just using the initial fixed list here
-	peerURLs := cls.PeerURLs()
+	if cfg.dir == "" {
+		cfg.dir = fmt.Sprintf("%v.etcdproxy", cfg.name)
+		log.Printf("no proxy data-dir provided, using default proxy data-dir ./%s", cfg.dir)
+	}
+	err = os.MkdirAll(cfg.dir, 0700)
```

Comment:
i feel this is stupid to add. since it will be everywhere. And when I was watching the go team coding, they just use 0700.